app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AmendUndertakingPage.scala.html

```diff
@@ -0,0 +1,119 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.eusubsidycompliancefrontend.controllers
+@import uk.gov.hmrc.eusubsidycompliancefrontend.config.AppConfig
+@import uk.gov.hmrc.eusubsidycompliancefrontend.services.Journey
+@import uk.gov.hmrc.eusubsidycompliancefrontend.models.types._
+@import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.Sector.Sector
+@import uk.gov.hmrc.eusubsidycompliancefrontend.models.ContactDetails
+
+
+@this(
+        layout: Layout,
+        formHelper: FormWithCSRF,
+        button: components.Button,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios : GovukRadios,
+        govukSummaryList : GovukSummaryList
+)
+
+
+@(name: UndertakingName, sector: Sector, contacts: ContactDetails, previous: Journey.Uri)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(
+    pageTitle = Some(messages("undertaking.amendUndertaking.title")),
+    backLinkEnabled = true,
+    backLink = Some(previous)
+) {
+
+    @defining(
+        contacts match {
```

Comment:
I've seen something like this somewhere else and thought a list with flatten might clean things up, something like

```
List(contacts.phone, contacts.mobile).flatten.mkString("<br>")
```

Reply:
yeah I just did copy paste from some other file :), will try to do that 👍

Reply:
@sam-miele your suggested changes also included [here](https://github.com/hmrc/eu-subsidy-compliance-frontend/pull/37/commits/ee95b46ec1db4d1e2d64b452a7a7cf962e15df9b)

Reply:
whoops!!! forgot about test cases, let me push those changes too